### PR TITLE
feat(client): use the progress indicator when importing the dataset (#1736) 

### PR DESCRIPTION
**Pull request by andre-code**
### Description
This PR is to use the progress indicator component when the dataset import process is running.

####  Import the dataset into the existing project
add dataset existing project progress indicator

####  Import the dataset into a new project
add dataset new project progress indicator

/deploy renku-core=develop renku=tests-new-project-form-changes #persist

### CODE REVIEW

File: client/src/dataset/addtoproject/addDatasetStatus.js [hunk 1]
```diff
@@ -58,6 +59,18 @@ function AddDatasetStatus(props) {
     case "inProcess" :
       statusProject = <div><Loader size="14" inline="true" /> {text}</div>;
       break;
+    case "importing" :
+      statusProject = (
+        <ProgressIndicator
+          type={ProgressType.Indeterminate}
+          style={ProgressStyle.Dark}
+          title={"Importing Dataset..."}
+          description="We've received your project information. This may take a while."
```
Comment (ciyer) [inline client/src/dataset/addtoproject/addDatasetStatus.js:68]:
```suggestion
          description="Project is being created, afterwards the data set will be imported into it."
```

**Review by ciyer — CHANGES_REQUESTED** [review]
Small suggestion for change in a string.

File: client/src/dataset/addtoproject/addDatasetStatus.js (hunk 1 above)
Comment (andre-code) [reply client/src/dataset/addtoproject/addDatasetStatus.js]: Suggestion included!

**Review by ciyer — APPROVED** [review]
👍

**Pull request merged**